gi-gtk-declarative-app-simple/gi-gtk-declarative-app-simple.cabal

```diff
@@ -23,17 +23,19 @@ library
   exposed-modules:      GI.Gtk.Declarative.App.Simple
   build-depends:        base                    >=4.10    && <5
                       , async
-                      , gi-gobject              >= 2      && <3
+                      , gi-gobject
                       , gi-glib
-                      , gi-gtk                  >= 3      && <4
+                      , gi-gtk <4
                       , gi-gdk
-                      , haskell-gi              >= 0.21   && <0.24
-                      , haskell-gi-base         >= 0.21   && <0.24
-                      , haskell-gi-overloading  == 1.0
-                      , pipes                   >= 4      && <5
-                      , pipes-concurrency       >= 2      && <3
+                      , haskell-gi
+                      , haskell-gi-base
+                      , haskell-gi-overloading
+                      , pipes
+                      , pipes-concurrency
                       , text
-                      , gi-gtk-declarative      >= 0.4  && <0.7
+                      , gi-gtk-declarative
```

Comment:
So, really when I look at the diff here with my cleaner changes it looks like I just need to extend the upper bounds.  I still want to increase the lower bounds too though, since it won't build with some of the current lower bounds.  I am not sure about conventions for that, but at least it should be easier to start from the current work (which builds) and start adding the bounds in again.